### PR TITLE
8251377: [macos11] JTabbedPane selected tab text is barely legible

### PR DESCRIPTION
**Pull request by prsadhuk**
Please review a fix for jdk16.
On macOS 11 (bigsur), using the Swing Aqua Look and Feel, the text of the selected JTabbedPane tab title text is just a light gray outline of white text on a white background. The macOS 11 design inverted from dark background / light text to light background / dark text, so white text on white background is not legible.
Correct system color to use for this scenario, as per Apple, is [NSColor controlTextColor]
so the proposed fix is to use this system color for BigSur. 
For preBigSur releases, "white" is still used as the above color is for text color but the tabPane background color is still not readable through any Apple API, so [NSColor controlTextColor] which returns black will not be legible on preBigSur releases which has black background.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8251377](https://bugs.openjdk.java.net/browse/JDK-8251377): [macos11] JTabbedPane selected tab text is barely legible

### Reviewers
 * [Phil Race](https://openjdk.java.net/census#prr) (@prrace - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk16 pull/65/head:pull/65`
`$ git checkout pull/65`

### CODE REVIEW

File: src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java [hunk 1]
```diff
@@ -869,7 +870,7 @@ public Object createValue(UIDefaults defaultsTable) {
             "TabbedPane.tabsOverlapBorder", Boolean.TRUE,
             "TabbedPane.selectedTabTitlePressedColor", selectedTabTitlePressedColor,
             "TabbedPane.selectedTabTitleDisabledColor", selectedTabTitleDisabledColor,
-            "TabbedPane.selectedTabTitleNormalColor", selectedTabTitleNormalColor,
+            "TabbedPane.selectedTabTitleNormalColor", System.getProperty("os.version").contains("10.16") ? selectedControlTextColor : selectedTabTitleNormalColor,
```
Comment (prrace) [inline src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java]: That doesn't look very robust. What will happen on the next version of macOS ?

**Comment by prrace** [comment]
> _Mailing list message from [Alan Snyder](mailto:javalists@cbfiddle.com) on [swing-dev](mailto:swing-dev@openjdk.java.net):_
> 
> What alternative are you proposing? I don?t believe there is any option that is guaranteed not to require changing
> for some future release of macOS without some new API from Apple, e.g. ?selectedTabTextColor?.

Alan,
We contacted Apple and this is their recommendation since there is no way to future-proof it.

File: src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java (hunk 1 above)
Comment (prsadhuk) [reply src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java]: Check for BigSur and above is added

**Comment by prsadhuk** [comment]
Headful jtreg and jck tests have passed.

**Review by prrace — APPROVED** [review]

File: src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java [hunk 2]
```diff
@@ -870,7 +870,7 @@ public Object createValue(UIDefaults defaultsTable) {
             "TabbedPane.tabsOverlapBorder", Boolean.TRUE,
             "TabbedPane.selectedTabTitlePressedColor", selectedTabTitlePressedColor,
             "TabbedPane.selectedTabTitleDisabledColor", selectedTabTitleDisabledColor,
-            "TabbedPane.selectedTabTitleNormalColor", System.getProperty("os.version").contains("10.16") ? selectedControlTextColor : selectedTabTitleNormalColor,
+            "TabbedPane.selectedTabTitleNormalColor", JRSUIUtils.isMacOSXBigSurOrAbove() ? selectedControlTextColor : selectedTabTitleNormalColor,
```
Comment (mrserb) [inline src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java:873]: I guess an intention was to use the JRSUIUtils.isBigSurOrAbove field not the JRSUIUtils.isMacOSXBigSurOrAbove() method, otherwise, why did you add it.
